src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -99,27 +100,37 @@
                      dissoc
                      :collectible
                      :amount
-                     (when (= transaction-type :collecible) :tx-type))})))
+                     (when (= transaction-type :collectible) :tx-type))})))
 
-(rf/reg-event-fx :wallet/send-collectibles-amount
- (fn [{:keys [db]} [{:keys [collectible stack-id amount]}]]
-   {:db (-> db
-            (update-in [:wallet :ui :send] dissoc :token)
-            (assoc-in [:wallet :ui :send :collectible] collectible)
-            (assoc-in [:wallet :ui :send :tx-type] :collectible)
-            (assoc-in [:wallet :ui :send :amount] amount))
-    :fx [[:dispatch [:wallet/get-suggested-routes {:amount amount}]]
-         [:navigate-to-within-stack [:screen/wallet.transaction-confirmation stack-id]]]}))
+(rf/reg-event-fx
+ :wallet/set-collectible-to-send
+ (fn [{db :db} [{:keys [collectible current-screen]}]]
+   (let [one-collectible? (= (collectible.utils/collectible-balance collectible) 1)
+         collectible-tx   (-> db
+                              (update-in [:wallet :ui :send] dissoc :token)
+                              (assoc-in [:wallet :ui :send :collectible] collectible)
+                              (assoc-in [:wallet :ui :send :tx-type] :collectible))]
+     {:db (cond-> collectible-tx
+            one-collectible? (assoc-in [:wallet :ui :send :amount] 1))
+      :fx [(when one-collectible?
+             [:dispatch [:wallet/get-suggested-routes {:amount 1}]])
+           [:dispatch
+            [:wallet/wizard-navigate-forward
+             {:current-screen current-screen
+              :flow-id        :wallet-flow}]]]})))
```

Comment:
the idea here was to have the different flows marked.
i.e :asset-flow, :bridge-flow, :collectible-flow 👍 

I think we should use it this way and not mix the logic of these as they are all slightly different and it will be cleaner 👍

Reply:
It was due to the infra not supporting it. Answered in https://github.com/status-im/status-mobile/pull/19515#discussion_r1551949724